webapp/src/main/webapp/i18n/all.properties

```diff
@@ -922,3 +922,7 @@ minimum_hour=Invalid entry. Please specify at least an Hour.
 year_month=Invalid entry. Please enter a Year and Month.
 decimal_only=Invalid entry.  A decimal point is allowed, but thousands-separators are not.
 whole_number=Invalid entry. Enter a whole number with no decimal point or thousands-separators.
+
+
+label.dateTimeWithPrecision.year_capitalized = Year
```

Comment:
We should also see updates in the i18n files, no?
https://github.com/vivo-project/VIVO-languages/tree/master/webapp/src/main/webapp/i18n